app/Console/Commands/InventoryImporterCommand.php

```diff
@@ -70,9 +70,24 @@ public function handle(): void
                 $networkHost->connectionSuccessful();
                 ++$networkHostSuccessfulConnectionCount;
             } catch (UnexpectedClientResponseException $exception) {
+                $host = $networkHost->info->getInventoryUrl();
                 $networkHost->connectionFailed();
 
-                Log::error(sprintf('[Inventory Importer] %s', $exception->getMessage()));
+                $this->warn(sprintf(
```

Comment:
In the console scripts we should use the `LineFormatterTrait` trait to properly display logs directly on the console.
It displays in both places - console and a log file. It's especially useful for a developer to see directly logs instead of looking at a file.